src/Numerics/Providers/Compatibility.cs

```diff
@@ -0,0 +1,13 @@
+﻿#if NATIVE && NETSTANDARD1_3
+
+namespace MathNet.Numerics.Providers
+{
+    using System;
+
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+    internal class SuppressUnmanagedCodeSecurityAttribute : Attribute
+    {
+    }  
+}
+
+#endif
```

Comment:
We should add this file also to Numerics.csproj (which should include all cs files as it is the primary project which is manually maintained).